src/choose-playlists.js

```diff
@@ -0,0 +1,125 @@
+// Bandwidth to use in playlist selection when resolution information is not provided.
+// 0.5 MB/s chosen as it was the default VHS bandwidth to use for the start of the
+// rendition selection algorithm.
+export const DEFAULT_BANDWIDTH = 4194304;
```

Comment:
I actually just prepared a PR for it, but now I'm reconsidering. I think that, for now at least, it would be better to have the plugin act consistently, rather than relying on VHS' INITIAL_BANDWIDTH, which may change in the future.

Reply:
ok